openid-4-verifiable-presentations-1_0.md

```diff
@@ -460,6 +485,65 @@ To use `client_id_scheme` values `entity_id`, `did`, `verifier_attestation`, `x5
 
 Other specifications can define further values for the `client_id_scheme` parameter. It is RECOMMENDED to use collision-resistant names for such values.
 
+## Request URI Method POST {#request_uri_method_post}
+
+This request is handled by the Request URI endpoint of the Verifier.  
+
+The request MUST use the HTTP POST method with the https scheme, and the content type `application/x-www-form-urlencoded` and the accept header set to `application/oauth-authz-req+jwt`.
```

Comment:
because it follows the design of direct post (and the OAuth requests). I have filed an issue to have general discussion about form vs json encoded

see https://github.com/openid/OpenID4VP/pull/59#discussion_r1521452908